mi/common/zpls_plot.py

```diff
@@ -0,0 +1,178 @@
+"""
+@package mi.common
+@file mi/common/zpls_plot.py
+@author Rene Gelinas
+@brief ZPLSC Echogram generation for the ooicore
+
+Release notes:
+
+This class supports the generation of ZPLS echograms. It needs matplotlib version 1.3.1 for the code to display
```

Comment:
Please update requirements.txt to include the correct matplotlib version.

Reply:
Done.
Updated the text, "matplotlib version (or higher)".
Updated the requirements.txt to reflect the same.